frontend/messages/uk.json

```
@@ -2073,6 +2073,27 @@
 	"idpGroup": "Група IdP",
 	"idpGroups": "Групи IdP",
 	"idpGroupsDescription": "Увімкнути федерацію груп IdP (провізіонування SCIM, зіставлення груп IdP та членство користувачів у групах IdP).",
+	"ebiosRmStudyDomainHelpText": "Зміна домену переміщує дослідження та всі об'єкти його семінарів до вибраного домену.",
```

Biome (2.5.6)

[error] The key ebiosRmStudyDomainHelpText was already declared.

(lint/suspicious/noDuplicateObjectKeys)